app/controllers/api/v1/recordings_controller.rb

```diff
@@ -89,6 +89,24 @@ def recordings_count
         render_data data: count, status: :ok
       end
 
+      # GET/POST /api/v1/recordings/recording_url.json
+      def recording_url
```

Comment:
Shouldn't this API require a `ManageRecordings` permission on the given recording?

Reply:
Done